Prox/Prox/Data/TravelTimesProvider.swift

```diff
@@ -83,7 +83,7 @@ struct TravelTimesProvider {
     }
 
     static func canTravelFrom(fromLocation: CLLocationCoordinate2D, toLocation: CLLocationCoordinate2D, before: Date, withCompletion completion: @escaping (Bool) -> ()) {
-        let timeInterval = before.timeIntervalSince(Date())
+        let timeInterval = Date().timeIntervalSince(before)
```

Comment:
I don't think this is correct. From the docs:

> The interval between the receiver and the another parameter. If the receiver is earlier than anotherDate, the return value is negative. If anotherDate is nil, the results are undefined.

`before` is expected to be a Date in the future: (`canTravelFrom(fromLocation:toLocation:before:withCompletion`), which means in the new code, the return value will be negative.